bedrock/base/templates/includes/protocol/footer/firefox.html

```diff
@@ -0,0 +1,9 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% if LANG.startswith('en-') %}
```

Comment:
Some locales have already completed translations, we may be able to add a few to this conditional before it merges. I'll make a list.